docs/formulary/parameters.rst

```diff
@@ -4,9 +4,23 @@
 Plasma parameters (`plasmapy.formulary.parameters`)
 ***********************************************************
 
+.. currentmodule:: plasmapy.formulary.parameters
+
 .. automodapi:: plasmapy.formulary.parameters
    :no-heading:
 
+.. currentmodule:: plasmapy.formulary.aliases.parameters
+
+=======
+Aliases
+=======
+
+These are not star-imported when you do `from plasmapy.formulary.parameters import *` by default;
+you can do that through `from plasmapy.formulary.aliases.parameters import *`.
+
+.. automodsumm:: plasmapy.formulary.aliases.parameters
+
+
```

Comment:
For maximum DRY this could be a custom sphinx directive. But I'm not that kind of crazy.